test/html/test_html.py

```diff
@@ -197,8 +197,10 @@ def test_html_customize_ul(tmp_path):
 
     # 1. Customizing through class attributes:
     class CustomPDF(FPDF):
-        li_tag_indent = 5
-        ul_bullet_char = "\x86"
+        def __init__(self):
+            super().__init__()
+            self.li_tag_indent = 40
+            self.ul_bullet_char = "\x86"
```

Comment:
The intent behind that portion of code seems to be to utilise this portion of `FPDF.write_html()`:
```
        kwargs2 = vars(self)
        # Method arguments must override class & instance attributes:
        kwargs2.update(kwargs)
        html2pdf = self.HTML2FPDF_CLASS(self, *args, **kwargs2)
```
However, `vars()` does not return static attributes, meaning that previously the `li_tag_indent` and `ul_bullet_char` of the custom `FPDF` inheritor class did nothing. Now they are bound to individual objects and are returned by `vars()`.

Reply:
> That is a very weird piece of code there, and I'm not sure if it really still serves any purpose (if it ever has). As far as I'm aware, there's no relevant overlap between instance attributes of `FPDF()` and keyword parameters of `HTML2PDF()`.

Not sure what you mean here, but in case you are talking about whether or not the `self.li_tag_indent` and `self.ul_bullet_char` are correctly used in later calls of `pdf.write_html()`, then I did check that and this piece of code now works correctly. Previously, it did not, which I noticed after looking at the indentation of relevant lists in the resulting `.pdf` files.

> Most likely we could just eliminate kwargs2 there without anyone missing it, and I hope this is going to happen soon (edit: just created https://github.com/py-pdf/fpdf2/issues/1198 with that goal)

Should I leave this for other volunteers (at least for now in case nobody will take it)?

Reply:
> Not sure what you mean here

Well, passing arguments that way does technically work, just that it is a horrible idea in terms of software architecture.

> Should I leave this for other volunteers (at least for now in case nobody will take it)?

Before actually removing that "feature", let's see what kind of reactions #1198 will receive. But I do recommend to not use it in your tests, or someone will have to fix those later as well.

Reply:
Now that #1198 has been merged, first you'll need to rebase, and then I think you can just roll back your changes to this test (unless you're really adamant about having larger indents).